docs/customizing.md

```diff
@@ -159,9 +163,111 @@ stages:
 Note the installation hooks only cover installation procedures, for upgrades equivalent
 `before-upgrade` and/or `after-upgrade-chroot` should be defined.
 
+### Adding extra LVM volume group disks during the installation
+
+This example is covering the case the host has multiple disks, some of them used
+as part of an LVM setup.
+
+In this particular case we will assume a host includes three disks (`/dev/sda`, `/dev/sdb`
```

Comment:
We use vdX in the examples (see below comment)
```suggestion
In this particular case we will assume a host includes three disks (`/dev/vda`, `/dev/vdb`
```

Reply:
Already changed to `sd*` instead of `vd*`. I prefer using device names not suggesting libvirt use.